code-challenges.test.js

```diff
@@ -14,38 +14,143 @@
 
 // a) Create a test with an expect statement using the variable provided.
 
-const people = [
-  { name: "ford prefect", occupation: "a hitchhiker" },
-  { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
-  { name: "arthur dent", occupation: "a radio employee" }
-]
-// Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
+describe("betterThanGrammarly", () => {
+  it("takes in an array of objects and returns an array with a sentence about each person with their name capitalized", () => {
+      const people = [
+          { name: "ford prefect", occupation: "a hitchhiker" },
+          { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
+          { name: "arthur dent", occupation: "a radio employee" }
+        ]
+    expect(betterThanGrammarly(people)).toEqual(["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."])
+  })
+})
+
+//  ReferenceError: betterThanGrammarly is not defined
 
 
 // b) Create the function that makes the test pass.
 
+// const betterThanGrammarly = (array) => 
+//   return array.map =(({key1,key2}) => {
+//     return key1.charAt(0).toUpperCase() +
+//   })
+// }
+
+
+//Yall really know how to pick some hard prompts, cause boy..... this is a doozy. I've eaten three plates of cheez-its trying to figure this out. I'm stressed and eating my feelings. 
+
```

Comment:
I once said your Beyonce comment was the best comment ever.  I was wrong.  This is the best comment ever